airootfs/etc/profile

```diff
@@ -49,4 +49,10 @@ unset TERMCAP
 # Man is much better than us at figuring this out
 unset MANPATH
 
-/usr/share/vx-img/flash-image.sh
+SELENTRY=$(< /sys/firmware/efi/efivars/LoaderEntrySelected-4a67b082-0a4c-41cf-b6c7-440b29bb8c4f)
+
+if [[ $SELENTRY == *"02-archiso-x86_64-ram-linux.conf"* ]]; then 
+    /usr/share/vx-img/flash-image.sh
+elif [[ $SELENTRY == *"03-archiso-x86_64-ram-linux.conf"* ]]; then 
+    /usr/share/vx-img/scrape-image.sh
+fi
```

Comment:
this could use a comment explaining what's going on here. I imagine it's that we are being smart about boot entries for user simplicity, but worth clarifying.

Reply:
can do. Yeah, we're basically using different systemd-boot entries to control which script gets run, so we don't have to reimplement a menu.